Cargo.lock

```diff
@@ -887,7 +887,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "4114279215a005bc675e386011e594e1d9b800918cea18fcadadcce864a2046b"
 dependencies = [
  "borsh-derive 0.10.3",
- "hashbrown 0.12.3",
+ "hashbrown 0.13.2",
```

Comment:
This changed automatically since the crate specifies [`hashbrown = ">=0.11,<0.14"`](https://github.com/near/borsh-rs/blob/79097e3c71ae469a101b4828457792bcf8be7f5f/borsh/Cargo.toml#L25) version requirement, and I confirmed it's the same in the v1.18 branch:

https://github.com/anza-xyz/agave/blob/7696b27311660e6cbe68e8f4324bd72d242b0348/Cargo.lock#L934